sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/ServiceClient.cs

```diff
@@ -16,6 +17,8 @@ namespace Azure.AI.TextAnalytics.ServiceClients
     ///
     internal abstract class ServiceClient
     {
+        public abstract ClientDiagnostics Diagnostics { get; }
```

Comment:
Could you explain to me why the need for this?

Reply:
Because the LROs are using it, currently.   Generally, we don't want to allow access to underscore members; those are intended to be private to the class.   Once we need access outside, it's better to have a property or a field named like a property to send a clear signal that it's safe to use this thing.

Reply:
:| ugh right